### PR TITLE
chore: Fixing root pom

### PR DESCRIPTION
**Pull request by ddixit14**

### CODE REVIEW

**Review by suztomo — APPROVED** [review]
If some check fails, it means that you may need to modify the template of this pom.xml.

**Comment by suztomo** [comment]
"Generated Files Sync / root pom.xml does not have diff from generated one" failed. Fix the template as well.

**Comment by suztomo** [comment]
@ddixit14 Run the commands below:

```
bash generation/print_root_pom.sh > pom.xml
bash generation/apply_current_versions.sh
```

**Comment by ddixit14** [comment]
Thanks Tomo. It passed now. added the automerge label.

File: generation/parent.pom.xml [hunk 1]
```diff
@@ -22,15 +22,49 @@
...
+          <url>https://google.oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+          <id>sonatype-nexus-staging</id>
+          <url>https://google.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+        <site>
+          <id>github-pages-site</id>
+          <name>Deployment through GitHub's site deployment plugin</name>
+          <url>site/google-cloud-bom</url>
```
Comment (meltsufin) [inline generation/parent.pom.xml:49]: @suztomo I don't think this is correct for the monorepo.

Comment (ddixit14) [reply generation/parent.pom.xml:49]: Sorry, correcting this.

**Review by meltsufin — APPROVED** [review]

**Comment by ddixit14** [comment]
`Kokoro - Test: Integration` failed. A flake probably. Re-ran it.

**Pull request merged**